repo/fsrepo/defaultds.go

```diff
@@ -13,22 +14,30 @@ import (
 	repo "github.com/ipfs/go-ipfs/repo"
 	config "github.com/ipfs/go-ipfs/repo/config"
 	"github.com/ipfs/go-ipfs/thirdparty/dir"
+
+	multi "github.com/ipfs/go-ipfs/Godeps/_workspace/src/github.com/ipfs/go-datastore/multi"
+	filestore "github.com/ipfs/go-ipfs/filestore"
 )
 
 const (
 	leveldbDirectory = "datastore"
 	flatfsDirectory  = "blocks"
+	fileStoreDir     = "filestore-db"
+	fileStoreDataDir = "filestore-data"
 )
+const useFileStore = true
+
+var _ = io.EOF
 
-func openDefaultDatastore(r *FSRepo) (repo.Datastore, error) {
+func openDefaultDatastore(r *FSRepo) (repo.Datastore, *filestore.Datastore, error) {
```

Comment:
The useFileStore is a temporary hack.  I thought about making it a config var., but once a filestore is enabled in the repository, disabling it can cause problems, especially with the pinner.   I am still not sure but am not thinking that it should be enabled via a command.